kitsune/sumo/static/js/ui.js

```diff
@@ -6,6 +6,17 @@
   $(document).ready(function() {
     initFolding();
     initAnnouncements();
+    $(".delete-profile-btn").removeAttr('disabled');
+
+    $('#delete-profile').submit(function(event){
+      var username = prompt("Are you sure you want to close your account? You cannot undo this action. Please enter your username to proceed.");
```

Comment:
This string used to be localized, but it isn't anymore. You need to wrap this in a call to gettext (`_`) too,